### DISCUSSION TITLE
How does LTS version package update?

### DISCUSSION THREAD

**Discussion by Lewisyixin** [apache/apisix#7177] [Q&A] [answered]
2.13.1 is a LTS version. When new bug in 2.13.1 found and fixed, will you update package 2.13.1 or just solve bug in next version?

**Comment by tzssangglass** [comment, marked as answer]
During the maintenance period of LTS (6 months from release), all bug fixes will be backported to the LTS version.

**Reply by Lewisyixin** [reply]
That means when I use version 2.13.1 , if I meet a bug  which has already been fixed. I just need to excute rpm or yum command to install 2.13.1 version again?

**Reply by tzssangglass** [reply]
yes

**Reply by juzhiyuan** [reply]
BTW @Lewisyixin, welcome to share your upgrade process by Blog or Video Recording 😄 It's helpful to users.
